### ISSUE TITLE
Back to the original API?

### ISSUE THREAD

**Issue by AlexeyKupershtokh** [kerryknight/phonegap-revmob-plugin#2]
You write:

> I've updated how you start a RevMob session/initialize the SDK. See Installation step #3 below. All other function calls are the same as the official documentation.

I wonder what was the necessary to do so? If I developed the plugin initially I would try to go with the original API:
1) this allows using the original revmob documentation as well;
2) less own code to support;

**Comment by manuelpaulo** [comment]
Indeed. You should clearly state that the initialization should now be:

RevMob.initWithAppId("123456789");

And _not_ how the official docs say:

revmob = new RevMob("123456789");
